### PR TITLE
added CONTRIBUTING.md and ARTICLE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by jasondobry**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Contributing to Thermo
+
+## Overview
+Welcome to the Thermo Repo! Here, you will find everything you need to start contributing to the Thermo Library. All you need is a GitHub account and the ability to explain how to solve a problem. If we accept your submission, our editors will review your content and direct you to make necessary changes before merging your content with the Library.
+
+## What you need
+* Expertise in your topic
+* A tested, verified procedure or procedures for a topic related to Thermo
+* A GitHub account and a fork of the Thermo repo.
```
Comment (laszlof) [inline CONTRIBUTING.md]: This should probably say "thermo/docs repository"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+* [Crucial criteria](#crucial-criteria)
+* [Payments](#payments)
+* [Submission process](#submission-process)
+* [Style guide](#style-guide)
+* [Template](#template)
+* [Using images](#using-images)
+
+## Crucial criteria
+* **Accuracy:** Your procedures are straightforward, correct, and tested. For each action, include the “why” as well as the “what.” Explain the purpose behind each action.
+* **Completeness:** Your guide produces a complete, functional configuration. When appropriate, your guide also gives readers of what to do next and provides all critical elements of security and other best practices.
```
Comment (laszlof) [inline CONTRIBUTING.md]: Check the wording of this sentence.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+* **Originality:** Anything that is not raw code is original content written exclusively for Thermo.
+
+## Payments
+We dispense all payments via PayPal. In most cases, we process payment within 30 days of publication, though in some cases in may take longer.
+  Updates under 500 words (for example, CentOS 6 > 7) $100
```
Comment (laszlof) [inline CONTRIBUTING.md]: This whole section could be a markdown table for better readability.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+Much of this process will be familiar to anyone familiar with GitHub. This section outlines the entire process, from your pull request (PR) to publication.
+
+1. Submit your PR to the [thermo.io/docs master branch](https://github.com/thermoio/docs).
+2. One or more subject matter experts (SMEs) reviews your PR and evaluates it for accuracy and completeness. They may direct your to make changes to your PR that must be resolved before proceeding to Step 3, or they may reject it.
+3. The editor reviews your PR for adherence to our conventions and [Style Guide](#style-guide), as well as checks it for accuracy, completeness, and originality. The editor may direct you to make further changes that must be resolved before proceeding to Step 4.
+4. Our team conducts a final review. Upon acceptance, the editor sends you the IP Agreement in DocuSign.
+5. Upon completion of the IP Agreement, our team merges your PR and publishes your submission to the Thermo Library.
+
+## Style guide
+We advise all contributors to review our Style Guide before submitting PRs; otherwise, our SMEs or editor will direct you to make such changes and prolong the effort.
```
Comment (laszlof) [inline CONTRIBUTING.md]: Do we have a style guide yet? Perhaps this should link to the example article?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+  
+  1. Install ModSec
+     ```shell
+     sudo yum install mod_security
+     ```
+  2. Restart Apache
+     ```shell
+     sudo service httpd restart
+     ```
+### Dates
```
Comment (laszlof) [inline CONTRIBUTING.md]: new line before this.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+Before uploading any images to the Thermo Repo:
+1. Crop the image appropriately. Balance size with the need to guide the reader.
+2. Make minimal but effective use of arrows, highlights, blurs, and other graphical tools to direct the reader. 
+
+To embed an image in your article, create a file in the images folder in your fork, then submit this file along with your article in the PR.
+
+1. In your fork, `<your_fork>/docs/images`, create a file that matches your article's name without the `.md` extension.
+2. In that file, click **Upload files**, then upload the images.
+3. In that same file, select the iamge, then click **Download**. Copy the URL in your browser, which you will use in Step 4 for `<URL_for_image_download>`.
+4. In your article, use the following Markdown, but replace the angled brackets (`<>`) with the URL you copied in Step 3.
```
Comment (laszlof) [inline CONTRIBUTING.md]: No markdown for this?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+See [Article Template](link)
+
+## Using Images
+Before uploading any images to the Thermo Repo:
+1. Crop the image appropriately. Balance size with the need to guide the reader.
+2. Make minimal but effective use of arrows, highlights, blurs, and other graphical tools to direct the reader. 
+
+To embed an image in your article, create a file in the images folder in your fork, then submit this file along with your article in the PR.
+
+1. In your fork, `<your_fork>/docs/images`, create a file that matches your article's name without the `.md` extension.
```
Comment (laszlof) [inline CONTRIBUTING.md]: "create a file" should be "create a directory"

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+2. In that file, click **Upload files**, then upload the images.
```
Comment (laszlof) [inline CONTRIBUTING.md]: as above. directory

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+3. In that same file, select the iamge, then click **Download**. Copy the URL in your browser, which you will use in Step 4 for `<URL_for_image_download>`.
```
Comment (laszlof) [inline CONTRIBUTING.md]: directory.

**Review by laszlof — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (jasondobry) [reply CONTRIBUTING.md]: changed "gives readers of what do to next" to "gives readers ideas about what to do next..."

File: CONTRIBUTING.md (hunk 4 above)
Comment (jasondobry) [reply CONTRIBUTING.md]: Everything below that IS the Style Guide. THere is a Templates header sub-topic within that will point to the template, but we could pull that to the front of this section.

File: CONTRIBUTING.md (hunk 5 above)
Comment (jasondobry) [reply CONTRIBUTING.md]: added

File: CONTRIBUTING.md (hunk 7 above)
Comment (jasondobry) [reply CONTRIBUTING.md]: changed two occurrences of "create a file" to "create a directory."

File: CONTRIBUTING.md (hunk 8 above)
Comment (jasondobry) [reply CONTRIBUTING.md]: changed

File: CONTRIBUTING.md (hunk 9 above)
Comment (jasondobry) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 6 above)
Comment (jasondobry) [reply CONTRIBUTING.md]: added

**Review by jasondobry — COMMENTED** [review]
Made all changes

**Review by laszlof — APPROVED** [review]

**Pull request merged**